docs/method/retro/0021-two-way-github-sync/witness/verification.md

```
@@ -0,0 +1,36 @@
+---
+title: "Verification Witness for Cycle 21"
+---
+
+# Verification Witness for Cycle 21
+
+This witness proves that `Two-way GitHub Sync` now carries the required
+behavior and adheres to the repo invariants.
+
+## Test Results
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)